test/tests/transfers/transfer_once_per_fixed_point/tests

```diff
@@ -0,0 +1,41 @@
+[Tests]
+  issues = '#22143'
+  design = 'Transfers/index.md'
+  [transfer_on_fp_begin]
+    type = CSVDiff
+    input = parent.i
+    cli_args = 'MultiApps/active=sub MultiApps/sub/cli_args=Outputs/fp/execute_on=MULTIAPP_FIXED_POINT_END
+                Outputs/file_base=transfer_fp_begin Outputs/fp_end/file_base=transfer_fp_begin_on_fp_end'
+    csvdiff = 'transfer_fp_begin.csv transfer_fp_begin_on_fp_end.csv transfer_fp_begin_sub0_fp.csv'
+    requirement = "The Transfer system shall support execution at the beginning of the fixed point iterations."
```

Comment:
```suggestion
    requirement = "The system shall support transferring data between applications at the beginning of fixed point iterations."
```
Similarly for other requirements; keep implementation details out (any capital letters is usually a red flag for me)